### ISSUE TITLE
[DEV] auditlog table will be updated for each command as auditnosyslog is 0 by default

### ISSUE THREAD

**Issue by chenglch** [xcat2/xcat-core#864]
Currently, auditlog table keeps the record information for every command. As db update operation is slow, is it necessary to open auditlog feature by default?

**Comment by daniceexi** [comment]
My idea is to make the audilog to be disabled by default. If user wants use it, just enable it.

**Comment by whowutwut** [comment]
I also noticed that each entry in the audit log is a few lines for each command.  Is there a way to get it down to one line for each command? What was the reason behind that format? is it a standard?  It's hard to parse or grep the current format entry...

**Comment by daniceexi** [comment]
We can set the **site.auditskipcmds=ALL** by default to avoid the db writing for each command.

Refer to: https://sourceforge.net/p/xcat/wiki/Syslog_and_auditlog/#auditlog
